locale/en-US/tbSync.dtd

```diff
@@ -1,15 +1,15 @@
 <!ENTITY popup.opensettings "Open TbSync account manager">
 <!ENTITY menu.settingslabel "Synchronization Settings (TbSync)">
 
-<!ENTITY toolbar.label " Synchronize all TbSync accounts">
+<!ENTITY toolbar.label "Synchronize all TbSync accounts">
 <!ENTITY toolbar.tooltiptext "Synchronize latest changes">
 
 <!ENTITY manager.AccountActions "Account actions">
 <!ENTITY manager.AddAccount "Add new account">
 <!ENTITY manager.SyncAll "Synchronize all enabled accounts">
 
 <!ENTITY manager.SynchronizeAccount "Synchronize account">
-<!ENTITY manager.EnableAccount "Enable account &amp; try to connect to server">
+<!ENTITY manager.EnableAccount "Enable account and try to connect to server">
```

Comment:
I want to keep the &

Reply:
english poor using ampersand instead of _and_